feature-libs/cart/import-export/components/import-to-cart/import-entries-dialog/import-entries-dialog.component.ts

```diff
@@ -14,21 +20,28 @@ import {
 } from '@spartacus/storefront';
 import { BehaviorSubject, Observable } from 'rxjs';
 import { finalize, pluck } from 'rxjs/operators';
-
 @Component({
   selector: 'cx-import-entries-dialog',
   templateUrl: './import-entries-dialog.component.html',
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class ImportEntriesDialogComponent {
+export class ImportEntriesDialogComponent implements OnDestroy {
```

Comment:
it seems as unused (`ngOnDestroy(): void {}`) so you can revert this changes

Reply:
done